docker/docker-compose.yaml

```diff
@@ -0,0 +1,170 @@
+version: '3'
+services:
+    zookeeper:
+        image: wurstmeister/zookeeper
+        networks:
+            - backend
+        expose:
+            - "2181"
+        restart: unless-stopped
+    kafka:
+        image: wurstmeister/kafka:0.10.2.0
+        environment:
+            KAFKA_ADVERTISED_HOST_NAME: kafka
+            KAFKA_ADVERTISED_PORT: "9092"
+            KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock
+        networks:
+            - backend
+        expose:
+            - "9092"
+        depends_on:
+            - zookeeper
+        restart: unless-stopped
+    postgres:
+        image: postgres:9.6
```

Comment:
Would it make sense to use the alpine linux version to avoid pulling in a full ubuntu install?
postgres:9.6-alpine does the trick for me.